### DISCUSSION TITLE
Why Golang ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#604] [General]
Hi everyone,

I see that this project is distributed like this at the moment:
![2022-01-25-112915_](https://user-images.githubusercontent.com/23012674/150961932-193e4373-de06-47c8-81f4-e451b33d7c5a.png)

I don't know if with github we can see the history of that language distribution over time..?

But For what I can see in is that the golang seem to grew is this project..

Why ? 

It's worries me when I compare the license of [Python](https://en.wikipedia.org/wiki/Python_(programming_language)) Vs [Golang](https://en.wikipedia.org/wiki/Go_(programming_language)) ( That even have a patent... and was created at g00gle ![Gosh_nervous](https://user-images.githubusercontent.com/23012674/150962790-34add974-73a7-4c4f-b306-e4f22ef1f1a1.gif) )

**Comment by gustavo-iniguez-goya** [comment]
The daemon and the GUI are 2 different applications, even if both are in the same repository. Github just count all the lines and classifies them.

On the other hand, the daemon is coded in Golang. We have added more functionality in the last 3 years, so is normal that it has grown.

Regarding the licenses, as far as I can tell both are FSF approved licenses ([BSD-3](https://directory.fsf.org/wiki/License:BSD-3-Clause))
